engine/src/main/java/org/terasology/engine/logic/console/ConsoleImpl.java

```diff
@@ -63,13 +63,17 @@ public void registerCommand(ConsoleCommand command) {
         Name commandName = command.getName();
 
         if (commandRegistry.containsKey(commandName)) {
-            logger.warn("Command with name '{}' already registered by class '{}', skipping '{}'",
-                    commandName, commandRegistry.get(commandName).getSource().getClass().getCanonicalName(),
-                    command.getSource().getClass().getCanonicalName());
+            logger.atWarn().
+                    addArgument(() -> commandName).
+                    addArgument(() -> commandRegistry.get(commandName).getSource().getClass().getCanonicalName()).
+                    addArgument(() -> command.getSource().getClass().getCanonicalName()).
+                    log("Command with name '{}' already registered by class '{}', skipping '{}'");
         } else {
             commandRegistry.put(commandName, command);
-            logger.debug("Command '{}' successfully registered for class '{}'.", commandName,
-                    command.getSource().getClass().getCanonicalName());
+            logger.atDebug().
+                    addArgument(() -> commandName).
+                    addArgument(() -> command.getSource().getClass().getCanonicalName()).
+                    log("Command '{}' successfully registered for class '{}'.");
```

Comment:
revert changes and suppress PMD `GuardLogStatement` warnings via method annotation